tests/resources/technical-records.json

```diff
@@ -7,7 +7,7 @@
     "vin": "XMGDE02FS0H012345",
     "partialVin": "012345",
     "trailerId": "09876543",
-    "systemNumber": "10000001",
+    "systemNumber": "11000001",
```

Comment:
these changes seem arbitrary, and may well mess up the automation testers. Why are you making these changes?

Reply:
On this branch I am calling the test-number microservice to generate the systemNumber. System Number generation starts from 10000000 and because we already have systemNumbers starting from 10000001, when I call the generateSystemNumber function it will generate systemNumbers that we already have in the seed file and all the POST int tests will fail. 

The solution that I thought of was to change the system numbers in the seed file and also the ones in test-results so sync these records (I have to create another PR for it), but if you say it will have an impact on automation we might need to consider removing the POST int tests. I'm open to other suggestion of course

Reply:
I would also add that now if we want to run the integration tests on tech-records we need to run locally the test-number microservice, otherwise the int tests that are calling test-number microservice will fail.

Reply:
oh dear